# sensor config
camera_config:
    X_offset: -300 # mm
    Y_offset: 60 # mm
    FOV: 90 # degrees
    image_height: 785 # px
    image_width: 785 # px
  
# vision neural network models
models:
    # object detector
    yolo:
        opencv:
            yolo_weights_file: '/home/putm/dv_ws/src/putm_dv_vision/config/yolov4-tiny_608-3l_dv_final.weights'
            yolo_config_file: '/home/putm/dv_ws/src/putm_dv_vision/config/yolov4-tiny_608-3l_dv.cfg'
            confidence_threshold: 0.5
            nms_threshold: 0.3
            model_size: 416
        tensorrt:
            model_path: '/home/putm/dv_ws/src/putm_dv_vision/config/yolov4.trt'
    # 2D to 3D projection
    rektnet:
        onnx:
            model_path: '/home/putm/dv_ws/src/putm_dv_vision/config/rektnet.onnx'
        tensorrt:
            model_path: '/home/putm/dv_ws/src/putm_dv_vision/config/rektnet.trt'